_config.yml

```diff
@@ -16,6 +16,14 @@ collections:
   tools:
     output: true
     permalink: /tools/:title/
+  team-members:
+    order:
+      - victoria.md
+      - eric.md
+      - akke.md
+      - marc.md
+      - chinari.md
```

Comment:
I think we should rename `Chinari` to `Subhechha`.

Reply:
done !